src/System.Private.Uri/tests/FunctionalTests/IriTest.cs

```diff
@@ -64,32 +64,35 @@ public void Iri_Uri_ShouldNotThrowArgumentOutOfRange()
         public void Iri_Uri_SchemaParsing_ShouldNotThrowArgumentOutOfRange()
         {
             string root = "viewcode://./codeschema_class?";
-            string uriDataFra = root + Uri.EscapeDataString("Type=\u00E9");
+            string uriDataFra = root + "Type=" + Uri.EscapeDataString("\u00E9");
 
             Uri u1 = new Uri(uriDataFra);
 
-            // TODO #8330 : Normalization should produce the same result for escaped/unescaped URIs.
-            // Assert.Equal(root + "Type=%C3%A9", u1.AbsoluteUri);
-
-            Assert.NotEqual(root + "Type=%C3%A9", u1.AbsoluteUri);
+            Assert.Equal(root + "Type=%C3%A9", u1.AbsoluteUri);
```

Comment:
You flipped it from NotEqual to Equal ... what has changed in code that it was passing and is still passing after your change. The test will run on all platforms - Core, Desktop and UWP, right?

Reply:
The change on line 67 allowed me to swap it from NotEqual to Equal. In the previous implementation of the test the = sign in "Type=" was being encoded along with the Unicode character. Since = is a reserved character it was left encoded in the normalized version of the URI, as it should be. I could have also left line 67 as is and changed 71 to 
```
Assert.Equal(root + "Type%3D%C3%A9", u1.AbsoluteUri);
```
for the same effect. Ultimately my goal here was to make the test more specific about the desired outcome than just that it is not equal to one possible output. It isn't a must have change if you'd rather it be left as-is.

Reply:
I was suspicious that it is the = ... I just didn't think we would have such Assert in place in such case.
Your version is much better, thanks!